client/layout/masterbar/masterbar-notifications/notifications-bell-icon.tsx

```diff
@@ -13,20 +13,22 @@ export const BellIcon: React.FC< Props > = ( { active } ) => (
 				fill="var( --color-masterbar-text )"
 				d="M9.9,20h4c0,0.5-0.2,1-0.6,1.4   c-0.8,0.8-2,0.8-2.8,0C10.1,21,9.9,20.5,9.9,20z M20,17.5v1H4v-1l0.9-0.7C5.5,16.3,6,15.5,6,15l0-5.5c0-3.3,2.7-6,6-6   c3.3,0,6,2.7,6,6V15c0,0.5,0.5,1.4,1.1,1.8L20,17.5z"
 			/>
-			<g className="notifications-bell-icon__bubble">
-				<use
-					// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-					className="border"
-					xlinkHref="#bubble"
-					transform="scale( 1.5 )"
-					fill={
-						active
-							? 'var( --color-masterbar-item-active-background )'
-							: 'var( --color-masterbar-background )'
-					}
-				/>
-				<use xlinkHref="#bubble" fill="var( --color-masterbar-unread-dot-background )" />
-			</g>
+			{ newItems && (
+				<g className="notifications-bell-icon__bubble">
+					<use
+						// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+						className="border"
+						xlinkHref="#bubble"
+						transform="scale( 1.5 )"
+						fill={
+							active
+								? 'var( --color-masterbar-item-active-background )'
+								: 'var( --color-masterbar-background )'
+						}
+					/>
```

Comment:
I think you don't need this if you use the proper path for the bell, as in the chipped vs the full path based on `newItems` boolean.

Reply:
It sounds like you are suggesting I go back to using two different SVG's and switch between the with vs without bubble depending on the `newItems`?

The problem with that is the bubble is really difficult to animate. If you animate the bubble it does not inherently have the border around it so the bubble is pulsing while the chip is not. But maybe I am missing something? I tried to add stroke but it never looked quite right.

As for maintaining the illusion of transparency, if the fill is using the same variable as the CSS is using for the background there shouldn't be any discrepancy. I did just notice that I had a typo and used the wrong variable so it is slightly off by a shade. I can update that.

Reply:
I see now. It didn't occur to me that we want to increase the gap size as we grow the bubble during the animation. Let's fix the color and move on.